### PR TITLE
Add step for build go-common image

### PR DESCRIPTION
**Pull request by Goginet**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -41,9 +40,12 @@ jobs:
 
       # Label used to access the service container
       postgres:
-        image: postgres
+        image: cr.webdevelop.us/webdevelop-pro/postgres:latest
```
Comment (vladyslav2) [inline .github/workflows/ci.yaml:43]: @Goginet we need to have a clear separation been common modules and specific projects. Here we should not use anything from invest project, and why would we need to do it anyway? Why cannot we use just a postgres? Do we need all those tables?

File: docker/go.mod [hunk 2]
```diff
@@ -0,0 +1,86 @@
+module github.com/webdevelop-pro/gcp
+
+go 1.21
```
Comment (vladyslav2) [inline docker/go.mod:3]: Should be go 1.22

File: docker/go.mod [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+toolchain go1.22.1
```
Comment (vladyslav2) [inline docker/go.mod:5]: This need to be removed

File: docker/readme.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Go lang common image
+
+Pre-build docker image for go 1.21.5
```
Comment (vladyslav2) [inline docker/readme.md:3]: go 1.22

**Review by vladyslav2 — CHANGES_REQUESTED** [review]
Let's separate invest project from go common

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Goginet) [reply .github/workflows/ci.yaml]: @vladyslav2 убрал, там был один тест который требовал этого, я его подправил и поменял на обычный образ postgres

File: docker/go.mod (hunk 2 above)
Comment (Goginet) [reply docker/go.mod]: @vladyslav2 done

**Review by vladyslav2 — APPROVED** [review]

**Pull request merged**